### ISSUE TITLE
Support binding to generic typed parameter in action handler method argument list

### ISSUE THREAD

**Issue by mailtous** [actframework/actframework#413]
Java code:

```
    @GetAction("user/page")
    public RenderAny page(Page<User> page) throws SQLException {
        String sql = " select * from user where id > ?";
        page = userService.getPage(page, sql, new Object[]{0});
        page.setUrl("page");
        render(page);
        return tpl("user_page.html");
    }
```

It will report error with the above code

**Comment by greenlaw110** [comment]
泛型本身是支持的,比如 List<String> 等等, 需要看看你的具体情况

**Comment by mailtous** [comment]
@greenlaw110
可以的，按上面的例子运行就会报错。

**Comment by greenlaw110** [comment]
@mailtous please provide the following information:

* the source code of `Page` class
* the detailed error stack

BTW your code has a little problem or code smell:

These 2 lines of code:

```
render(page);
return tpl("user_page.html");
```

better be simplified to:

```
return template("user_page.html", page);
```

**Comment by mailtous** [comment]
Page的原码在下面link.
其实只要随便建一个带泛型的VO，比如Foo<T> 再用( Foo foo) 做为入参,ACT就会启动时直接报错。很容易重现的。
https://github.com/mailtous/act-eagle-allone/blob/master/src/main/java/com/artlongs/framework/page/Page.java

**Comment by mailtous** [comment]
Foo<T>

**Comment by mailtous** [comment]
"Foo<T>"

**Comment by mailtous** [comment]
Foo<T>

**Comment by greenlaw110** [comment]
Fixed in 1.7.0
